charts/application-core/templates/success-rate.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.rollouts.enabled }}
+{{- if .Values.rollouts.successRateTemplate.enabled }}
+---
+apiVersion: argoproj.io/v1alpha1
+kind: AnalysisTemplate
+metadata:
+  name: {{ .Release.Name }}-success-rate
+spec:
+  args:
+    - name: deployment
+    - name: latest-hash
+  metrics:
+    - name: success-rate
+      interval: {{ .Values.rollouts.successRateTemplate.interval  }}
+      successCondition: {{ .Values.rollouts.successRateTemplate.successCondition }}
+      count: {{ .Values.rollouts.successRateTemplate.count }}
+      failureLimit: {{ .Values.rollouts.successRateTemplate.failureLimit }}
+      consecutiveSuccessLimit: {{ .Values.rollouts.successRateTemplate.consecutiveSuccessLimit }}
+      provider:
+        prometheus:
+          # TODO maybe make this an arg?
+          address: http://prometheus.linkerd-viz.svc.cluster.local:9090
```

Comment:
yea, make that an arg b/c that prom doesnt' exist in the DEV cluster anymore. And I'll start working on INT next week